microprofile/5.0/opentracing/3.0/22.0.0.1-beta-TCKResults.adoc

```diff
@@ -0,0 +1,37 @@
+:page-layout: certification
+= TCK Results
+
+As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License], following is a summary of the TCK results for releases of MicroProfile Interoperable JWT RBAC 2.0.
+
+== Open Liberty 22.0.0.1-beta - MicroProfile Interoperable JWT RBAC 2.0 Certification Summary
+
+* Product Name, Version and download URL (if applicable):
++
+https://repo1.maven.org/maven2/io/openliberty/openliberty-runtime/22.0.0.1-beta/openliberty-runtime-22.0.0.1-beta.zip[Open Liberty 22.0.0.1-beta]
+* Specification Name, Version and download URL:
++
+link:https://download.eclipse.org/microprofile/microprofile-opentracing-2.0/microprofile-opentracing-spec-2.0.html[MicroProfile OpenTracing 2.0]
```

Comment:
Again, I think this should be 3.0-RC4 but I can only find a link for RC3
https://download.eclipse.org/microprofile/microprofile-opentracing-3.0-RC3/microprofile-opentracing-spec-3.0-RC3.html
```suggestion
link:https://download.eclipse.org/microprofile/microprofile-opentracing-3.0-RC4/microprofile-opentracing-spec-3.0-RC4.html[MicroProfile OpenTracing 3.0-RC4]
```

Reply:
https://search.maven.org/search?q=g:org.eclipse.microprofile.opentracing%20AND%20a:microprofile-opentracing-api%20AND%20v:3.0-RC4

Reply:
@Emily-Jiang Same as my previous comment ... 

That is a link to an index page for the API. What we need is a link to the spec (in html). If you look here - https://download.eclipse.org/microprofile/ - there is no `microprofile-opentracing-3.0-RC4`